protocol/txsender/sender.go

```diff
@@ -42,10 +42,11 @@ type privateKeySender struct {
 func NewSenderWithPrivateKey(ctx context.Context, ec *ethclient.Client, contract *contracts.SwapFactory,
 	erc20Contract *contracts.IERC20, txOpts *TxOpts) Sender {
 	return &privateKeySender{
-		ctx:      ctx,
-		ec:       ec,
-		contract: contract,
-		txOpts:   txOpts,
+		ctx:           ctx,
+		ec:            ec,
+		contract:      contract,
```

Comment:
Now that we have 2 contracts, should this one be renamed to something like `swapContract`?

Reply:
yeah, done!